### PR TITLE
PCF-439 Serve imgs from project src vs github

### PR DESCRIPTION
**Pull request by Carla-Moz**
This PR closes issue [PCF-439](https://mozilla-hub.atlassian.net/browse/PCF-438)

[PCF-439]: https://mozilla-hub.atlassian.net/browse/PCF-439?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by julienw — COMMENTED** [review]
This looks good to me, but there's another file to be imported :-) See comments below

File: src/resources/Strings.tsx [hunk 1]
```diff
@@ -45,9 +45,6 @@ export const Strings = {
       base: {
         title: 'Compare with a base',
         tagline: 'Analyze differences between specific revisions.',
-        img: 'https://user-images.githubusercontent.com/88336547/233237125-1534220b-c343-421a-9133-ce8f151cb979.png',
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply src/resources/Strings.tsx:48]: Ahh the header. Thanks for catching.

File: src/styles/CompareCards.ts [hunk 2]
```diff
@@ -1,6 +1,10 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```
[the remark this answers is not in the document]
Comment (Carla-Moz) [reply src/styles/CompareCards.ts]: Resolved the warning

**Review by julienw — APPROVED** [review]
Thanks!

**Pull request merged**